docs/rmlint.1.rst

```diff
@@ -153,28 +153,31 @@ General Options
 
     Choose the algorithm to use for finding duplicate files. The algorithm can be
     either **paranoid** (byte-by-byte file comparison) or use one of several file hash
-    algorithms to identify duplicates.  The following well-known algorithms are available:
+    algorithms to identify duplicates.  The following cryptographic algorithms are available:
```

Comment:
Not sure if it's of any interest to the user if a hash algorithm is suitable for crypotgraphic usage in the context of ``rmlint```. The only property is that the hash function produces well distributed results. 

Sidenote: I'm not entirely sure if ``Highway`` is a "good" cryptographic hash function. I never heard it in that context,
nor do I read a clear statement about it in the project readme. I could be totally wrong here though.

Reply:
Yes the highway documentation suggests some cryptographic properties but falls short of making any strong claims.  There is a good article at https://arxiv.org/pdf/1612.06257.pdf (you can skip to the conclusions) that is a bit more explicit - I like the "To the best of our (non-expert) ability, we have analyzed the algorithm and
found no weaknesses"

I think all of our hashes have been shown to give well distributed results, so for a given hash length they should all have similar probability of random collisions.  For resistance to malicious collisions I would expect blake and sha3 to be equivalent to each other and better than the others.  But I guess the best anyone can say is "no known attack that can craft a malicious hash collision".

Anyway will update the manpage to avoid unsupported claims.